config.tf

```diff
@@ -57,20 +57,21 @@ variable "tectonic_container_images" {
   type        = "map"
 
   default = {
-    addon_resizer                        = "gcr.io/google_containers/addon-resizer:2.1"
-    bootkube                             = "quay.io/coreos/bootkube:v0.10.0"
-    machine_config_operator              = "docker.io/openshift/origin-machine-config-operator:v4.0.0"
-    etcd_cert_signer                     = "quay.io/coreos/kube-etcd-signer-server:678cc8e6841e2121ebfdb6e2db568fce290b67d6"
-    etcd                                 = "quay.io/coreos/etcd:v3.2.14"
-    hyperkube                            = "openshift/origin-node:latest"
-    kube_core_renderer                   = "quay.io/coreos/kube-core-renderer-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
-    kube_core_operator                   = "quay.io/coreos/kube-core-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
-    tectonic_channel_operator            = "quay.io/coreos/tectonic-channel-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
-    kube_addon_operator                  = "quay.io/coreos/kube-addon-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
-    tectonic_alm_operator                = "quay.io/coreos/tectonic-alm-operator:v0.3.1"
-    tectonic_ingress_controller_operator = "quay.io/coreos/tectonic-ingress-controller-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
-    tectonic_utility_operator            = "quay.io/coreos/tectonic-utility-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
-    tectonic_network_operator            = "quay.io/coreos/tectonic-network-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
+    addon_resizer                                  = "gcr.io/google_containers/addon-resizer:2.1"
+    bootkube                                       = "quay.io/coreos/bootkube:v0.10.0"
+    machine_config_operator                        = "docker.io/openshift/origin-machine-config-operator:v4.0.0"
+    etcd_cert_signer                               = "quay.io/coreos/kube-etcd-signer-server:678cc8e6841e2121ebfdb6e2db568fce290b67d6"
+    etcd                                           = "quay.io/coreos/etcd:v3.2.14"
+    hyperkube                                      = "openshift/origin-node:latest"
+    kube_core_renderer                             = "quay.io/coreos/kube-core-renderer-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
+    kube_core_operator                             = "quay.io/coreos/kube-core-operator-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85"
+    openshift_service_serving_cert_signer_operator = "openshift/origin-service-serving-cert-signer:latest"
```

Comment:
this stands out as weird looking.

Reply:
If the variable name here does not need to match the url then it could be shortened to openshift_service_ca_operator, and we can just update the url to point to origin-service-ca when we are ready to do that.

Reply:
Renamed to "openshift_service_ca_operator" and "openshift_service_ca_operator_image"